### PR TITLE
Add inst stage to bring us to 38% sites can give flood status

### PR DESCRIPTION
**Pull request by lindsaycarr**
Related to #98. Due to missing daily stage (and missing NWS flood stage), only 17% of gages could actually indicate flood stage. The code here calculates mean daily stage using instantaneous stage for gages that had a non-missing daily flow and a flood stage (so those that had the potential to show us flood). It takes ~2 hours to run this code. 

This brings us from 17% of gages being able to indicate flood status to 38%!

### CODE REVIEW

File: 1_fetch.yml [hunk 1]
```diff
@@ -98,6 +102,16 @@ targets:
   1_fetch/out/dv_data.rds:
     command: gd_get('1_fetch/out/dv_data.rds.ind')
 
+  # -- calculate dv stage for gages missing it using inst --
```
Comment (aappling-usgs) [inline 1_fetch.yml]: could you define "gh" in this comment?

File: 1_fetch/src/fetch_inst_stage.R [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+#' @param dv_data_ind indicator file for the data.frame of dv_data
+#' @param stage_data_ind indicator file for the data.frame of sites with flood stage
+#' @param request_limit number indicating how many sites to include per dataRetrieval request (from viz_config.yml)
+fetch_inst_stage <- function(ind_file, dv_data_ind, stage_data_ind, request_limit){
+
+  dv_data <- readRDS(scipiper::sc_retrieve(dv_data_ind, remake_file = '1_fetch.yml'))
+  sites_stage <- readRDS(scipiper::sc_retrieve(stage_data_ind, remake_file = '1_fetch.yml'))
+
+  dv_data <- readRDS("1_fetch/out/dv_data.rds")
+  sites_stage <- readRDS("1_fetch/out/sites_stage.rds")
```
Comment (aappling-usgs) [inline 1_fetch/src/fetch_inst_stage.R]: it looks like maybe lines 12 and 13 were shortcuts during development but should be removed in favor of lines 9 and 10 now

File: 1_fetch/src/fetch_inst_stage.R [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+      data_i <- data_i %>%
+        dataRetrieval::renameNWISColumns() %>%
+        mutate(day = as.Date(dateTime))
+
+      if("GH_Inst" %in% names(data_i)) {
+        gh_mean <- data_i %>%
+          group_by(site_no, day) %>%
+          summarize(GH_mean = mean(GH_Inst, na.rm = TRUE))
+      } else {
+        gh_mean <- gh_mean <- data_i %>%
```
Comment (aappling-usgs) [inline 1_fetch/src/fetch_inst_stage.R]: double assignment to gh_mean?

File: viz_config.yml [hunk 4]
```diff
@@ -71,7 +72,7 @@ title_cfg:
   subtitle: "Water Year 2018"
   footnote: [
     "Conditions are relative to the historic daily record for each gage.",
-    "Flood status was readily available for 17% of sites when this graphic was produced."
+    "Flood status available for 38% of sites when this graphic was produced."
```
Comment (aappling-usgs) [inline viz_config.yml]: :+1: nice and concise. I might include the "was" just so the style is consistent with the other footnote.

File: 1_fetch/src/fetch_inst_stage.R (hunk 2 above)
Comment (lindsaycarr) [reply 1_fetch/src/fetch_inst_stage.R]: oh oops!

File: 1_fetch/src/fetch_inst_stage.R (hunk 3 above)
Comment (lindsaycarr) [reply 1_fetch/src/fetch_inst_stage.R]: copy/paste error...

**Pull request merged**